vmm_core/virt/src/x86/vp.rs

```diff
@@ -1818,6 +1852,7 @@ state_trait! {
     (11, "cet", cet, set_cet, Cet),
     (12, "cet_ss", cet_ss, set_cet_ss, CetSs),
     (13, "tsc_aux", tsc_aux, set_tsc_aux, TscAux),
+    (14, "pat", pat, set_pat, Pat),
```

Comment:
Maybe reorder+renumber these, too. This is a breaking change (but it doesn't matter since this state isn't stable yet), so you might as well get it the way we want.